### ISSUE TITLE
demo code

### ISSUE THREAD

**Issue by sssssshf** [eternaldolphin/LaMI-DETR#6]

**Comment by sssssshf** [comment]
请问  什么时候开源demo的code？ 是推理的demo吗

**Comment by eternaldolphin** [comment]
Yes, but I'm currently busy with my thesis. If you're in a hurry, you can contact me via email dupenghui@buaa.edu.cn . Thank you for your understanding.

**Comment by sssssshf** [comment]
> Yes, but I'm currently busy with my thesis. If you're in a hurry, you can contact me via email [dupenghui@buaa.edu.cn](mailto:dupenghui@buaa.edu.cn) . Thank you for your understanding.

大佬推理demo还没出吗

**Comment by eternaldolphin** [comment]
> > Yes, but I'm currently busy with my thesis. If you're in a hurry, you can contact me via email [dupenghui@buaa.edu.cn](mailto:dupenghui@buaa.edu.cn) . Thank you for your understanding.
> 
> 大佬推理demo还没出吗

push了，看最新一次PR。
得到visual desc的system prompt可以自己调调看，不一定是最优解，包括用什么api也可以试试。
训推一致是最好的，不一致也能出结果
